Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/threshold/ThresholdedRandomCutForest.java

```diff
@@ -203,20 +241,43 @@ public ThresholdedRandomCutForest(Builder<?> builder) {
         }
         timeStampDeviation = new Deviation(discount);
         forest = builder.buildForest();
+
+        if (normalizeTime || normalizeValues) {
+            int number = forest.getOutputAfter() + forest.getShingleSize() - 1 + ((differencing) ? 1 : 0);
+            startNormalization = builder.startNormalization.orElse(number);
+            checkArgument(startNormalization >= forest.getShingleSize() + ((differencing) ? 1 : 0),
+                    " use startNormalization() with n at least" + (forest.getShingleSize() + ((differencing) ? 1 : 0)));
+            stopNormalization = builder.stopNormalization.orElse(DEFAULT_STOP_NORMALIZATION);
+            checkArgument(startNormalization <= stopNormalization, "normalization stops before start");
+            initialValues = new double[startNormalization][];
+            initialTimeStamps = new long[startNormalization];
+        }
+
         BasicThresholder basic = new BasicThresholder(builder.anomalyRate);
         if (forest.getDimensions() / forest.getShingleSize() == 1) {
-            basic.setLowerThreshold(1.1);
-            basic.setHorizon(0.75);
+            basic.setLowerThreshold(BasicThresholder.DEFAULT_LOWER_THRESHOLD_ONED);
+            basic.setHorizon(BasicThresholder.DEFAULT_HORIZON_ONED);
         }
         thresholder = basic;
     }
```

Comment:
How do you set the horizon and lowerThreshold from the ThresholdedRandomCutForest builder?

Reply:
Not feasible in this PR. Fixing. In general if the numeric parameters of BasicThresholder has to be changed from the larger class then the current interface is inappropriate because the interface should now have all the variable setters/getters in the same class. It also may make sense to be able to change these dynamically on the fly.